sdcm/sct_events/database.py

```diff
@@ -81,57 +67,22 @@ def add_info(self: T_log_event, node, line: str, line_number: int) -> T_log_even
         return super().add_info(node=node, line=line, line_number=line_number)
 
 
-DatabaseLogEvent.add_subevent_type("WARNING", severity=Severity.WARNING,
+DatabaseLogEvent.add_subevent_type("WARNING", severity=Severity.SUPRESS,
```

Comment:
so we won't see Scylla Warnings as Warning events anymore?

Reply:
yep, they would be in the logs, but we won't be collecting them as events.

since we don't have any usages of those events,and their numbers in some cases can be quite big

we still have warning events that can be raise by SCT itself.